File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/CQS/BalancesFunction.vb
Imports System
Imports System.Threading.Tasks
Imports System.Collections.Generic
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.Contracts.CQS
Imports Nethereum.ABI.FunctionEncoding.Attributes
Imports StandardToken.MyContractName.DTOs
Namespace StandardToken.MyContractName.CQS

    <[Function]("balances", "uint256")>
    Public Class BalancesFunction
        Inherits ContractMessage
    
        <[Parameter]("address", "", 1)>
        Public Property ReturnValue1 As String
    
    End Class


End Namespace

File: src/Nethereum.Generators.JavaScript/codeGenNodeTests/MyContractName/CQS/BalancesFunction.cs
using System;
using System.Threading.Tasks;
using System.Numerics;
using Nethereum.Hex.HexTypes;
using Nethereum.Contracts.CQS;
using Nethereum.ABI.FunctionEncoding.Attributes;
using StandardToken.MyContractName.DTOs;
namespace StandardToken.MyContractName.CQS
{
    [Function("balances", "uint256")]
    public class BalancesFunction:ContractMessage
    {
        [Parameter("address", "", 1)]
        public string B {get; set;}
    }
}
